langchain/chains/moderation.py

```diff
@@ -91,6 +88,6 @@ def _call(
         run_manager: Optional[CallbackManagerForChainRun] = None,
     ) -> Dict[str, str]:
         text = inputs[self.input_key]
-        results = self.client.create(text)
+        results = self.client.create(text, self.model_name, self.openai_api_key)
```

Comment:
need to pass organization?

Reply:
was model name just not used before?

Reply:
> 

It seems that there is no such parameter.
https://github.com/openai/openai-python/blob/main/openai/api_resources/moderation.py#L28